content/blogposts/2024/top-5-tips-for-using-cody-with-react.md

```diff
@@ -0,0 +1,306 @@
+---
+title: "Top 5 Tips For Using Cody With React"
+authors:
+  - name: Deepak Kumar
+    url: https://twitter.com/deepakdk3478
+  
+publishDate: 2024-01-09T10:00-07:00
+description: "In this blog, Deepak Kumar has shown how Cody uses the context by building an React application. He has also mentioned few tips of using Cody to get the best result."
+tags: [blog]
+slug: "top-5-tips-for-using-cody-with-react"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/og-top-5-tips-for-using-cody-with-react.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/og-top-5-tips-for-using-cody-with-react.png
+---
+
+<Alert type="secondary">This is a guest blog post from our community. <a href="https://discord.com/servers/sourcegraph-969688426372825169" target="_blank">Join our Discord</a> to meet the author and connect with other developers passionate about Code Search and AI!</Alert>
+
+React is a popular frontend library used for building fast interactive user interfaces. React is a common choice for most organizations in terms of a frontend library. Also, as different LLM models and ChatGPT are here, developers are using them a lot to answer coding questions and write some code. Taking into account how AI helps to boost the productivity of the software developer, many coding AI assistants have been developed, and [Cody](https://about.sourcegraph.com/cody) is one of them created by Sourcegraph. 
```

Comment:
Can we link React to https://react.dev/ 

```suggestion
React is a popular frontend library used for building interactive user interfaces. React is a common choice for many organizations. 
```
I would add 2-3 more sentences here about why companies use React as opposed to talking about AI and LLMs.